packages/vscode-extension/src/debugging/DebugSession.ts

```diff
@@ -35,37 +39,13 @@ export class DebugSession implements Disposable {
     });
   }
 
-  public dispose() {
-    this.vscSession && debug.stopDebugging(this.vscSession);
-    this.debugEventsListener.dispose();
-  }
-
-  public async start() {
-    const websocketAddress = await this.metro.getDebuggerURL();
-    if (!websocketAddress) {
-      return false;
-    }
-
-    let sourceMapAliases: Array<[string, string]> = [];
-    const isUsingNewDebugger = this.metro.isUsingNewDebugger;
-    if (isUsingNewDebugger && this.metro.watchFolders.length > 0) {
-      // first entry in watchFolders is the project root
-      sourceMapAliases.push(["/[metro-project]/", this.metro.watchFolders[0]]);
-      this.metro.watchFolders.forEach((watchFolder, index) => {
-        sourceMapAliases.push([`/[metro-watchFolders]/${index}/`, watchFolder]);
-      });
-    }
-
+  public async startDebugSession() {
```

Comment:
we are mixing up names here which makes this change very confusing. Sometime we refer to the connection with JS debugger as JSDebugger and at other times we use `DebugSession`. I think we should just keep using `start` here and in places where we explicitely deal with the CDP JS debugger, we should use `JSDebugger`